cmake/external_or_find_package.cmake

```diff
@@ -37,13 +37,16 @@ function(external_or_find_package LIB_NAME)
         set(${LIB_NAME}_INTERNAL FALSE)
     endif()
 
+    if (${LIB_NAME}_INTERNAL)
```

Comment:
exlude from all is needed only for external linking. I miss a "NOT" here.

Reply:
My bad... It should be excluded from all, because we do not want to pull this dependency by default. Only if other target pulls in on demand.